src/annotations/AnnotationParser.php

```diff
@@ -13,6 +13,10 @@
 
 namespace mindplay\annotations;
 
+if (!defined('T_TRAIT')) {
+    define('mindplay\\annotations\\T_TRAIT', -2);
```

Comment:
Why you're not defining a constant in global namespace like `define('T_TRAIT', -2);`?